project/config/communityrelations/config/pathauto.pattern.board_member_pattern.yml

```diff
@@ -1,24 +1,22 @@
-uuid: d9faca1d-4501-4265-bf78-ea5e46143288
+uuid: 69374ccb-2bf2-479c-ac4d-dbcb2f09bd38
 langcode: en
 status: true
 dependencies:
   module:
     - node
-_core:
-  default_config_hash: IPBus0EzmucmooeZ0Uit-0IpbgDhfBhvfMgthjcZmRs
 id: board_member_pattern
 label: 'Board member pattern'
 type: 'canonical_entities:node'
-pattern: 'board-members/[node:title]'
+pattern: '[node:title]'
```

Comment:
I think this pattern should be /about-us/board/[node:title]

Reply:
Done thank you Adam